backend/frontend/templates/frontend/user/board/js_exec_action_card_moved.html

```diff
@@ -0,0 +1,5 @@
+{% include "frontend/user/board/partial_card.html" %}
+
+<script>
```

Comment:
Hmm.. same comment as above. Also there is already a file that does this, now the code isn't DRY.